test/extended/syncer/syncer_utils.go

```diff
@@ -170,5 +170,9 @@ func (s *SyncTarget) Delete(k *exutil.CLI) {
 
 // Clean the SyncTarget resource
 func (s *SyncTarget) Clean(k *exutil.CLI) error {
+	// TODO: Temp solution for known limitation: https://github.com/kcp-dev/kcp/issues/999
+	// Need to remove after the issue solved
+	mySyncerKey, _ := k.WithoutNamespace().WithoutKubeconf().WithoutWorkSpaceServer().Run("get").Args("--server="+s.WorkSpaceServer, "synctarget", s.Name, `-o=jsonpath={.metadata.labels.internal\.workload\.kcp\.dev/key}`).Output()
+	k.AsPClusterKubeconf().WithoutNamespace().WithoutWorkSpaceServer().Run("delete").Args("ns", "-l", "internal.workload.kcp.dev/cluster="+mySyncerKey).Execute()
```

Comment:
Due to https://github.com/kcp-dev/kcp-tests/blob/main/test/extended/util/client.go#L748 checks `!c.asPClusterKubeconf`, `WithoutWorkSpaceServer()` can be removed.  
```suggestion
	k.AsPClusterKubeconf().WithoutNamespace().Run("delete").Args("ns", "-l", "internal.workload.kcp.dev/cluster="+mySyncerKey).Execute()
```

Reply:
Addressed.